tests/setup_cfg_fmt_test.py

```diff
@@ -227,6 +228,9 @@ def test_normalize_lib(lib, expected):
     ),
 )
 def test_adds_long_description_with_readme(filename, content_type, tmpdir):
+    dir_name = os.path.splitext(os.path.basename(filename))[0]
+    if dir_name != filename:  # we expect to ignore non file matches
+        tmpdir.join(dir_name).mkdir()
```

Comment:
I don't see how this is ever hit, generally you should not have logic in tests this should be a separate test

Reply:
It's hit here https://github.com/asottile/setup-cfg-fmt/blob/master/setup_cfg_fmt.py#L95, if you can make the CI coverage requirement without this I'm happy to drop it.

Reply:
I'm not exactly sure how this actually works, wouldn't it try and write contents to a directory a few lines down?

this should be a separate test probably -- could even be a unit test for `_first_files` -- I wonder why coverage has suddenly started worrying about this line though 🤔